network_modeling/configs/templates/dellmellanox/full/sw-leaf.secondary.j2

```diff
@@ -0,0 +1,73 @@
+{#- sw-leaf.secondary #}
+
+{%- include 'common/unknown.j2' %}
+hostname {{ variables.HOSTNAME }}
+
+rest api restconf
+
+load-balancing ingress-port enable
+no load-balancing mac-selection destination-mac
+no load-balancing mac-selection ethertype
+no load-balancing mac-selection source-mac
+no load-balancing mac-selection vlan-id
+no load-balancing tcp-udp-selection l4-destination-port
+no load-balancing tcp-udp-selection l4-source-port
+enhanced-hashing resilient-hashing ecmp
+
+
+spanning-tree mode mst
+spanning-tree port type edge default
+spanning-tree mst revision 1
+spanning-tree mst name MST0
+
+ntp enable
+ntp server {{ variables.NCN_W001 }}
+ntp server {{ variables.NCN_W002 }}
+ntp server {{ variables.NCN_W003 }}
+ntp source vlan2
+
+iscsi target port 860
+iscsi target port 3260
+
+
+router ospf 1
+router-id {{ variables.LOOPBACK_IP }}
+
+{% include 'tds/leaf-bmc-to-spine.lag.j2' %}
+{% include 'common/bmc.j2' %}
+
+interface loopback 0
+    ip address {{ variables.LOOPBACK_IP }}
```

Comment:
You'll need to add `/32` to all loopbacks

Aruba example
https://github.com/Cray-HPE/canu/blob/develop/network_modeling/configs/templates/full/sw-leaf-bmc.j2#L38